chainer_compiler/chainer_compiler.py

```diff
@@ -238,10 +238,8 @@ def __init__(self, model, onnx_file, used_translator, dump_onnx=False,
 
     def compile(self, onnx_file):
         # TODO(hamaji): Revive shape inference.
```

Comment:
Could you move this comment to line 262 again?

Reply:
fixed.